gp-includes/router.php

```diff
@@ -125,6 +125,10 @@ private function default_routes() {
 			"post:/$set/import-translations" => array( 'GP_Route_Translation', 'import_translations_post' ),
 			"post:/$set/-discard-warning" => array( 'GP_Route_Translation', 'discard_warning' ),
 			"post:/$set/-set-status" => array( 'GP_Route_Translation', 'set_status' ),
+
+			'post:/notes/-new'                             => array( 'GP_Route_Note', 'new_post' ),
+			'post:/notes/-edit'                            => array( 'GP_Route_Note', 'edit_post' ),
+			'post:/notes/-delete'                          => array( 'GP_Route_Note', 'delete_post' ),
```

Comment:
These lines are added in the middle of the set routes, move them further down to avoid any confusion.